src/ActiveSetMethods.jl

```diff
@@ -7,6 +7,8 @@ import MathOptInterface
 const MOI = MathOptInterface
 const MOIU = MathOptInterface.Utilities
 
+export statistics
```

Comment:
Let's not export this. It won't be used frequently. The name is too generic to be exported.